lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -174,7 +174,7 @@ bool CPUBackend::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
     }
   }
 
-  return true;
+  return elementTy == ElemKind::FloatTy;
```

Comment:
This means CPUBackend only support either float or int8(quantized type) right? But
CPU backend support int32 quantization: `int32_t libjit_element_quantize_kernel_i32`, I think  `isOpSupported(Kinded::Kind::QuantizeNodeKind, ElemKind::Int32QTy)` should be true... Or this call is illegal?

Reply:
Ah, good catch -- yes, `isOpSupported(Kinded::Kind::QuantizeNodeKind, ElemKind::Int32QTy)` should return true.